tests/test_automatic_fields.py

```diff
@@ -15,26 +17,27 @@ def name():
         return "tap_tester_bing_ads_minimum_fields_test"
 
     def expected_sync_streams(self):
-        # BUG_SRCE-4313 To reproduce grep jira id across tests dir and comment/uncomment correpsonding lines.
-        #               You will need to comment out all _report streams prior to running test as well.
-        #               You will probably get an OperationError during the sync, that's because we aren't selecting any fields
-        #               prior to the sync, this means only automatic fields will be replicated. Except we aren't marking the
+        # BUG_SRCE-4313 To reproduce grep jira id across tests dir and comment/uncomment
+        #               corresponding lines. You will need to comment out all _report streams prior
+        #               to running test as well. You will probably get an OperationError during the
+        #               sync, that's because we aren't selecting any fields prior to the sync, this
+        #               means only automatic fields will be replicated. Except we aren't marking the
         #               necessary fields as automatic. Hence this bug.
         return {
             'accounts',
-            'ad_extension_detail_report',
-            'ad_group_performance_report',
+            # 'ad_extension_detail_report',
+            # 'ad_group_performance_report',
             'ad_groups',
-            'ad_performance_report',
+            # 'ad_performance_report',
             'ads',
-            'age_gender_audience_report',
-            'audience_performance_report',
-            'campaign_performance_report',
+            # 'age_gender_audience_report',
+            # 'audience_performance_report',
+            # 'campaign_performance_report',
```

Comment:
Why don't we use streams_to_exclude here ? 
I also see this in another test

Reply:
because we hope to run new ads and get data for these reports in the future.  The exclusion might be temporary.